taf/api/keystore.py

```diff
@@ -85,3 +85,40 @@ def generate_keys(keystore: Optional[str], roles_key_infos: str) -> None:
                     rsa_key = keys.generate_rsa_key(role.length)
                     private_key_val = rsa_key["keyval"]["private"]
                     print(f"{role.name} key:\n\n{private_key_val}\n\n")
+
+
+@log_on_start(INFO, "Generating .taf directory", logger=taf_logger)
+def create_taf_directory():
+    # Create the .taf directory
+    taf_directory = Path(".taf")
+    taf_directory.mkdir(exist_ok=True)
+
+    # Create the config.toml file
+    config_file_path = taf_directory / "config.toml"
+    config_file_path.touch()  # Create an empty file
+
+    # Create the keystore directory
+    keystore_directory = taf_directory / "keystore"
```

Comment:
Haven't tested in detail, but one thing that should also be done is to set the keystore path to this location if the keystore path is not specified (this should be the case for all commands where the keystore is used). That means that it will be necessary to look for this `.taf` directory. We can expect to find it inside the library root.